app/transport/http/user.go

```diff
@@ -16,32 +15,37 @@ func (uh UserHandler) CreateUser() http.Handler {
 		user := entities.User{}
 
 		if err := json.NewDecoder(req.Body).Decode(&user); err != nil {
-			WriteJSONResponse(w, http.StatusBadRequest, Response{MsgBadRequest})
+			WriteJSONResponse(w, http.StatusBadRequest, Response{MsgBadRequest}, uh.logger)
+			uh.logger.Errorf("%+v\n", err)
```

Comment:
It's not enough to log only error. It would be cool to have additional information about execution context.
@andstepko, logging guru, could you please share some examples of good messages?

Reply:
@vpakhuchyi @andstepko  We could start with something simple like `logger.Errorf("JSON decode failed: %+v\n", err)`